waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/PanopticConcurrentOperationExecutor.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (C) 2016-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.mapping.service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.hotels.bdp.waggledance.mapping.model.DatabaseMapping;
+import com.hotels.bdp.waggledance.mapping.service.requests.RequestCallable;
+
+public class PanopticConcurrentOperationExecutor implements PanopticOperationExecutor {
+
+  private static final String INTERRUPTED_MESSAGE = "Execution was interrupted: ";
+  private static final String SLOW_METASTORE_MESSAGE = "Metastore {} was slow to respond so results are omitted";
+
+  private final static Logger LOG = LoggerFactory.getLogger(PanopticConcurrentOperationExecutor.class);
```

Comment:
LOG -> log please (since it's not immutable)

Reply:
We use LOG everywhere in WD, don't mind changing but I think better to stick with the same

Reply:
Ah, OK, I lose track of which projects we do which in. Agree, fine for another PR to make it consistent everywhere, I can just picture @AnanaMJ loving that one!